WalletWasabi.Backend/Controllers/BlockchainController.cs

```diff
@@ -424,15 +425,17 @@ public async Task<IActionResult> GetUnconfirmedTransactionChainAsync([FromQuery,
 	{
 		try
 		{
+			var before = DateTimeOffset.UtcNow;
 			uint256 txId = new(transactionId);
 
 			var cacheKey = $"{nameof(GetUnconfirmedTransactionChainAsync)}_{txId}";
-
-			return await Cache.GetCachedResponseAsync(
+			var ret = await Cache.GetCachedResponseAsync(
 				cacheKey,
 				action: (string request, CancellationToken token) => GetUnconfirmedTransactionChainNoCacheAsync(txId, token),
 				options: UnconfirmedTrasanctionChainCacheEntryOptions,
 				cancellationToken);
+			RequestTimeStatista.Instance.Add("unconfirmed-transaction-chain", DateTimeOffset.UtcNow - before);
+			return ret;
```

Comment:
In the specific context of this issue this is useless because the problem was happening before and we are not using this call yet. However, for future reference it's nice to have

Reply:
Yes. ACK. I keep it.